src/app/activity/add-edit-activity/add-edit-activity.component.ts

```diff
@@ -102,6 +106,15 @@ export class AddEditActivityComponent implements OnInit, OnDestroy {
             this.loading = false;
             this._changeDetectorRef.detectChanges();
           });
+
+        this.notificationProjectService.getAll(1, 1000, '+name')
+          .takeUntil(this.ngUnsubscribe)
+          .subscribe((res3: any) => {
+            if (res3) {
+              this.projectNotifications = res3.data;
+            }
```

Comment:
I think these subscriptions are a bit incorrectly setup. See: https://stackoverflow.com/questions/45087291/unsubscribing-an-array-of-subscriptions